scalableminds/webknossos
# Allow to disable auto saving for layout configs

- [X] implement functionality
- [x] clean up UI

### URL of deployed dev instance (used for testing):
- https://optionallayoutautosaving.webknossos.xyz

### Steps to test:
- ensure that you have at least two layout presets
- disable auto-saving by clicking the "disconnect" item in the layout submenu (the tooltips don't work on the first hovers for me :/ antd is probably not meant for this combination of menu and tooltip)
- change the active layout (e.g., make a pane bigger)
- switch to the other layout
- switch to the changed layout (it should not contain the bigger pane)
- enable auto-saving and repeat the above steps (the layout change should survive the layout switching)

### Issues:
- fixes #3592

------
